### DISCUSSION TITLE
Airflow scheduled dag_run time changes if DAG is run manuallly

### DISCUSSION THREAD

**Discussion by miguelpromptly** [apache/airflow#25876] [Q&A]
Hi Airflow community,

I'm using Airflow 2.2.5 built from the official docker image with a Postgres database. As an example, say I have a DAG set to run daily at 2am. As long as I only turn it on a let it run automatically, the process works fine and the DAG runs daily at 2am as intended, but if for some reason I need to do a manual run, say at 8:30am, now on the next day (and all the following ones) the DAG will run automatically at 8:30am instead of the programmed 2am.

I'v tried using cron expressions instead of the `datetime.timedelta` for the `schedule_interval` parameter and even though in the UI it appears the DAG will run at 2am (despite the 11am manual run), the DAG actually only runs at 8:30am contrary to the UI indication.

Example (you can see the logical date and run_id being 2am, but the queued_at and start_date are actually 8:30am):
![image](https://user-images.githubusercontent.com/89162200/185937700-781a2cee-975e-469f-b28b-73e67d550cc6.png)

As anyone else noticed this behavior? And is there anything I can do to prevent the manual runs from interfering with the scheduled ones?

**Comment by potiuk** [comment]
When you use data interval then I think it will just continue with data interval. If you want to schedule the dag using cron you should I think create a new dag_id for that one as the next run for your previous "interval" schedule might be already scheduled. Airlfow does not handle all the changes in DAG schedule yet. Until we implement https://cwiki.apache.org/confluence/display/AIRFLOW/AIP-36+DAG+Versioning the recommended approach when you change schedule is to change dag_id making it effectively another DAG (you can also delete the old DAG and recreate it)
